modules/oadp-features.adoc

```diff
@@ -21,7 +21,7 @@ You must exclude Operators from the backup of an application for backup and rest
 
 
 Restore::
-You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the restored objects by namespace, PV, or label.
+You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the objects to be restored by namespace, PV, or label.
```

Comment:
Do we need the "to be restored" passive construct? I think "You can restore" covers the restoration operation. 

@mburke5678 . WDYT?

Reply:
You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the objects by namespace, PV, or label.

@stillalearner Is this phrasing OK with you?

Reply:
@dfitzmau 
Thank you. I updated it based on your suggestion.

Reply:
@CarmiWisemon Looks good.